src/smart-components/user/user.js

```diff
@@ -165,50 +204,55 @@ const User = ({
     { title: userExists ? username : intl.formatMessage(messages.invalidUser), isActive: true },
   ];
 
+  
+  
   return (
     <Fragment>
       {userExists ? (
-        <Stack>
-          <StackItem>
-            <TopToolbar paddingBottm={false} breadcrumbs={breadcrumbsList()}>
-              <TopToolbarTitle
-                title={username}
-                renderTitleTag={() =>
-                  user && !isLoading ? (
-                    <Label color={user?.is_active && 'green'}>{intl.formatMessage(user?.is_active ? messages.active : messages.inactive)}</Label>
-                  ) : (
-                    <Skeleton size="xs" className="rbac__user-label-skeleton"></Skeleton>
-                  )
-                }
-                description={intl.formatMessage(messages.userDescription, { username })}
-              />
-            </TopToolbar>
-          </StackItem>
-          <StackItem>
-            <Section type="content" id={'user-detail'}>
-              <TableToolbarViewOld
-                columns={columns}
-                isCompact={false}
-                isExpandable={true}
-                onExpand={onExpand}
-                createRows={createRows}
-                data={roles.data}
-                filterValue={filter}
-                ouiaId="user-details-table"
-                fetchData={({ limit, offset, name }) => {
-                  debouncedFetch(limit, offset, name, ['groups_in'], username);
-                }}
-                setFilterValue={({ name }) => setFilter(name)}
-                isLoading={isLoading}
-                pagination={roles.meta}
-                filterPlaceholder={intl.formatMessage(messages.roleName).toLowerCase()}
-                titlePlural={intl.formatMessage(messages.roles).toLowerCase()}
-                titleSingular={intl.formatMessage(messages.role).toLowerCase()}
-                tableId="user"
-              />
-            </Section>
-          </StackItem>
-        </Stack>
+          <Stack>
```

Comment:
@JuliePrazakova can you please undo the indentation of that code so that the actual changes are visible?